src/main/java/at/hannibal2/skyhanni/config/features/gui/customscoreboard/CustomScoreboardConfig.java

```diff
@@ -33,7 +33,7 @@ public class CustomScoreboardConfig {
     @ConfigEditorDraggableList()
     public List<ScoreboardElement> scoreboardEntries = new ArrayList<>(ScoreboardElement.defaultOption);
 
-    @ConfigOption(name = "Reset Appearance", desc = "Reset the appearance of the advanced scoreboard.")
+    @ConfigOption(name = "Reset Appearance", desc = "Reset the appearance of the advanced scoreboard.\n§cYou will need to restart your game to apply the changes.")
     @ConfigEditorButton(buttonText = "Reset")
     public Runnable reset = () -> {
         scoreboardEntries = ScoreboardElement.defaultOption;
```

Comment:
Here, this assigns a the static list defaultOptions. But you need to clear the list and add in all elements from default options. This should fix it.